packages/core/core/src/ReporterRunner.js

```diff
@@ -60,9 +66,8 @@ export default class ReporterRunner {
           logger: new PluginLogger({origin: reporter.name}),
         });
       } catch (e) {
-        throw new ThrowableDiagnostic({
-          diagnostic: errorToDiagnostic(e, reporter.name),
-        });
+        // We shouldn't emit a report event here as we will cause infinite loops...
+        INTERNAL_ORIGINAL_CONSOLE.error(e);
```

Comment:
Maybe we should first try to emit the error as a reporter event and then see if it returns and log it into the original console if it persists?

Not entirely sure if it's really necessary, however logging the error straight to the original console would remove any fancy error formatting we do... for reporters